cp-quickstart/start-cloud.sh

```diff
@@ -93,13 +93,13 @@ retry $MAX_WAIT ccloud::validate_ccloud_ksqldb_endpoint_ready $KSQLDB_ENDPOINT |
 print_pass "Confluent Cloud KSQL is UP"
 
 printf "Obtaining the ksqlDB App Id\n"
-CMD="confluent ksql app list -o json | jq -r '.[].id'"
+CMD="confluent ksql cluster list -o json | jq -r '.[].id'"
```

Comment:
FYC: Now that we're using json, we can be more reliable with these commands. The array might contain more than one ksqldb app, and then we are storing the wrong app id and maybe causing issues for the user.  Instead we could be specific with `jq`...

```
confluent ksql cluster list -o json | jq --arg KSQLDB_ENDPOINT "$KSQLDB_ENDPOINT" -r '.[] | select(.endpoint==$KSQLDB_ENDPOINT) | .id'
```

Reply:
Point take.  In this case: if a user is running this script `cp-quickstart/start-cloud.sh`, it's a new environment so there is only one ksqlDB app.

Reply:
Yes, unless the user is running other commands during long scripting runs (like while the app provisions) and then results in confused state in the CLI.